src/instana/instrumentation/kafka/kafka_python.py

```diff
@@ -1,23 +1,31 @@
 # (c) Copyright IBM Corp. 2025
 
+
 try:
+    import contextvars
     import inspect
     from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple
 
     import kafka  # noqa: F401
     import wrapt
+    from opentelemetry import context, trace
     from opentelemetry.trace import SpanKind
 
     from instana.log import logger
     from instana.propagators.format import Format
+    from instana.singletons import get_tracer
     from instana.util.traceutils import (
         get_tracer_tuple,
         tracing_is_off,
     )
+    from instana.span.span import InstanaSpan
 
     if TYPE_CHECKING:
         from kafka.producer.future import FutureRecordMetadata
 
+    consumer_token: Dict[str, Any] = {}
```

Comment:
The same comment as [here](https://github.com/instana/python-sensor/pull/790/files#r2302190833).

BTW, there was too much repetitive code in those two files. It would be interesting to see how to reduce that.